gRPC-Swift-Plugins.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+
+    s.name = 'gRPC-Swift-Plugins'
+    s.version = '1.0.0-alpha.14'
+    s.license = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.summary = 'Swift gRPC code generator plugin binaries'
+    s.homepage = 'https://www.grpc.io'
+    s.authors  = { 'The gRPC contributors' => 'grpc-packages@google.com' }
+
+    s.source = { :http => "https://github.com/grpc/grpc-swift/releases/download/#{s.version}/protoc-swift-plugins-#{s.version}-macOS-x86_64.zip"}
```

Comment:
Sorry I think I've changed my position on this slightly: we need to wary of the fact that Macs on Apple Silicon are a thing. Either we need separate pods: or we should structure the zip appropriately and bundle them up together. What do you think @MrMage?

Reply:
I think including binaries for both architectures in the zip either in folders or as fat binaries if possible would be better than having separate pods.

Reply:
I will change the name of the zip again, then.

Reply:
(Unresolving since I'd like to hear @MrMage's opinion here)

Reply:
Full-fledged Mac app bundles can be distributed as a "fat" binary containing both architectures. Would that be a possibility here as well?

Reply:
When looking at the project generated by "make project" I would dare to say that this should be possible. Have not tried it in xcode12 though.

Reply:
Not sure since if that would work or not. I don't think the details are important at the moment though; the relevant question is do we distribute them in a single zip? It seems like we have consensus that we should.